etc/picongpu/taurus-tud/V100_picongpu.profile.example

```diff
@@ -1,76 +1,83 @@
+## `picongpu.profile` for the `ml` partition on taurus@TU-Dresden
+## The `ml` partition is equipped with Nvidia V100 GPUs
+## Nodes in the `ml` partition are Power9 systems which is different
+## from the headnodes. Therefore, compile on a node!
+
 # Name and Path of this Script ############################### (DO NOT change!)
 export PIC_PROFILE=$(cd $(dirname $BASH_SOURCE) && pwd)"/"$(basename $BASH_SOURCE)
 
-# User Information ######################################### (edit those lines)
+# User Information ######################################### (edit these lines)
 #   - automatically add your name and contact to output file meta data
 #   - send me a mail on batch system jobs: BEGIN, END, FAIL, REQUEUE, ALL,
 #     TIME_LIMIT, TIME_LIMIT_90, TIME_LIMIT_80 and/or TIME_LIMIT_50
 export MY_MAILNOTIFY="ALL"
-export MY_MAIL="someone@example.com"
+export MY_MAIL="none@no.ne"
 export MY_NAME="$(whoami) <$MY_MAIL>"
 
 # Text Editor for Tools ###################################### (edit this line)
 #   - examples: "nano", "vim", "emacs -nw", "vi" or without terminal: "gedit"
 #export EDITOR="nano"
 
-# Modules #####################################################################
+# Modules ##################################################
 #
 module purge
 module load modenv/ml
-# similar to foss/2018a, but also includes SpectrumMPI which basically is just a OpenMPI-fork of IBM
-module load gsolf/2018a
-module load GCC/6.4.0-2.28
-module load CMake/3.10.2-GCCcore-6.4.0
-# CUDA is no module in the current enviroment!
-#module load CUDA/9.2.88  # gcc <= 7, intel 15-17
-# OpenMPI is already loaded
-#module load OpenMPI/2.1.2-GCC-6.4.0-2.28
+
+# load CUDA/9.2.88-GCC-7.3.0-2.30, also loads GCC/7.3.0-2.30, zlib, OpenMPI and others
+module load fosscuda/2018b
+module load CMake/3.11.4-GCCcore-7.3.0
 module load git/2.18.0-GCCcore-6.4.0
-module load zlib/1.2.11-GCCcore-6.4.0
+module load libpng/1.6.34-GCCcore-7.3.0
 
-# Self-Build Software #########################################################
+printf "@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@\n"
+printf "@ Note: You need to compile picongpu on a node. @\n"
+printf "@       Similar for building the libraries.     @\n"      
+printf "@       Get a node with the getNode command.    @\n"
+printf "@       Then source %s again.@\n" "$(basename $PIC_PROFILE)"
+printf "@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@\n"
+
+# Self-Build Software #####################################
 #
 # needs to be compiled by the user
-export PIC_LIBS="$HOME/lib"
-export BOOST_ROOT=$PIC_LIBS/boost-1.69.0-Power9
-export PNG_ROOT=$PIC_LIBS/libpng-1.6.34-Power9
+# Check the install script at
+# (https://gist.github.com/steindev/cc02eae81f465833afa27fc8880f3473#file-picongpu_0-4-3_taurus-tud-sh)
```

Comment:
You might want to link that in the `docs/source/` as well, similar to how I do it on Piz Daint.

Reply:
Done

Reply:
Please without brackets, this also breaks the URL highlighting in some editors